### DISCUSSION TITLE
配置表里面的ip地址似乎被墙了

### DISCUSSION THREAD

**Discussion by FelixxChoi** [joyanhui/ikuai-bypass#43] [General]
ghp.ci似乎也被墙了。

**Comment by joyanhui** [comment]
配置文件里面的 github的镜像地址只是演示用，要稳定最好ikuai_bypass也在代理环境下使用。另外自己用cf搭建一个自己私有的镜像也是一个办法(我自己是这样用的)

**Comment by joyanhui** [comment]
另外一个方案是使用一些不知名的镜像站点，自己搜索一下
越出名的用的人越多越容易被墙
